jobs/config.json

```diff
@@ -635,6 +635,57 @@
       "sig-network"
     ]
   },
+  "ci-ingress-gce-e2e-release-1.0": {
```

Comment:
oh, no `.` in job names, as k8s cannot create pod with `.` in name

Reply:
Darn, is there a better alternative to "."?

I'd hate to have to say 'release-1-0"

Reply:
yep :-)

Reply:
Ok I made it so that it still uses the "." in the test grid name.